app/views/media_objects/_resource_description.html.erb

```diff
@@ -29,61 +29,88 @@ Unless required by applicable law or agreed to in writing, software distributed
 
   <%= render partial: 'text_field',
              locals: {form: form, field: :title,
-                      options: {required: true}} %>
+                      options: {label_parenthesis: t('metadata_parenthesis.title'),
+                                required: true}} %>
 
   <%= render partial: 'text_field',
-             locals: {form: form, field: :date_issued,
-                      options: {display_label: 'Publication date', required: true}} %>
+             locals: {form: form, field: :creator,
+                      options: {display_label: t('metadata_label.creator'),
+                                label_parenthesis: t('metadata_parenthesis.creator'),
+                                required: true,
+                                multivalued: true}} %>
 
   <%= render partial: 'text_field',
-             locals: {form: form, field: :creator,
-                      options: {display_label: 'Main contributor(s)', multivalued: true}} %>
+             locals: {form: form, field: :topical_subject,
+                      options: {display_label: t('metadata_label.topical_subject'),
+                                label_parenthesis: t('metadata_parenthesis.topical_subject'),
+                                required: true,
+                                multivalued: true}} %>
+
+  <%# TODO: This needs to be hooked up to AJAX, separate PR %>
```

Comment:
I'm currently working on the code that this comment refers to -- this PR could use a review.